app/src/organisms/ModuleWizardFlows/SelectLocation.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { css } from 'styled-components'
+import { getModuleDisplayName } from '@opentrons/shared-data'
+import {
+  RESPONSIVENESS,
+  TYPOGRAPHY,
+  SPACING,
+  SIZE_1,
+  Flex,
+} from '@opentrons/components'
+import { Banner } from '../../atoms/Banner'
+import { StyledText } from '../../atoms/text'
+import { GenericWizardTile } from '../../molecules/GenericWizardTile'
+import type { ModuleCalibrationWizardStepProps } from './types'
+
+export const BODY_STYLE = css`
+  ${TYPOGRAPHY.pRegular};
+
+  @media ${RESPONSIVENESS.touchscreenMediaQuerySpecs} {
+    font-size: 1.275rem;
+    line-height: 1.75rem;
+  }
+`
+
+export const SelectLocation = (
+  props: ModuleCalibrationWizardStepProps
+): JSX.Element | null => {
+  const { proceed, attachedModule } = props
+  const { t } = useTranslation('module_wizard_flows')
+  const moduleName = getModuleDisplayName(attachedModule.moduleModel)
+  // TODO: keep track of the selected slot in a state variable that can be
+  // used in calibration step
+  const handleOnClick = (): void => {
+    proceed()
+  }
+  const bodyText = (
+    <>
+      <Banner type="warning" size={SIZE_1} marginY={SPACING.spacing4}>
```

Comment:
slack (from live dqa)
https://opentrons.slack.com/archives/CSCLVUW3C/p1685644862441539

```suggestion
      <Banner type="warning" size="1rem" marginY={SPACING.spacing4}>
```